files.go

```diff
@@ -33,11 +36,15 @@ func (f *FilePool) Add(fileName string, r io.Reader) {
 		Panicf("")
 	}
 
-	b, err := ioutil.ReadAll(r)
+	b, err := io.ReadAll(r)
 	if err != nil {
 		Panicf("failed to output: %s", err)
 	}
 
+	if f.generatedHeader {
+		b = append([]byte(generatedComment), b...)
```

Comment:
I think we have to know the encoding to put that first, no?

Reply:
Yes, changed the interface instead in a way that doesn't cause any difference in usage. 🙂